regression-tests.auth-py/clientsubnetoption.py

```diff
@@ -148,7 +148,7 @@ def to_wire(self, file=None):
         file.write(data)
         return None
 
-    def from_wire(cls, otype, wire, current, olen):
+    def from_wire(self, otype, wire, current, olen):
```

Comment:
Ok, I think this is the correct change:
```suggestion
    @classmethod
    def from_wire(cls, otype, wire, current, olen):
```